Automattic/simplenote-android
# Remove sort bar in note lists and trash

Fixes #1342 #1358

### Fix
Removes the sort bar from both the search results and notes list.

### Test

1. Login with a valid account
2. Go to note list. The sort bar should not appear
3. Go to Trash list and the sort bar should not appear either
4. Go the Settings and change the sort settings
5. Come back to the note list and the notes should be sorted based on the setting set

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.


### Release
`RELEASE-NOTES.txt` was updated in 0eeacf0cf885231360849062e2cb97afe72c7cf0 with:
> Removed sort bar from notes list but still accessible from settings
